Docs/CHANGELOG.md

```diff
@@ -1,16 +1,22 @@
 # Changelog
 
-## 0.12.0 - Unreleased
+## 0.12.0
 
 ### Vision
+- Added detection of Japanese traffic signs
 - Added property `progress` to the `VisionReplayManager`
 - Added read-only property `duration` to the `VisionReplayManager`
+- Added replay support of recorded routes
```

Comment:
Could you please append this comment to the appropriate line in the AR section?
Like "Added ... method to the ... to support route replay from the recorded session".